vars/skipStage.groovy

```diff
@@ -178,9 +228,20 @@ boolean call(Map config = [:]) {
         case "Unit Test Bullseye":
             return skip_stage_pragma('bullseye', 'true')
         case "Unit Test with memcheck":
-            return skip_stage_pragma('unit-test-memcheck')
+            return ! params.CI_UNIT_TEST_MEMCHECK ||
+                   skip_stage_pragma('unit-test-memcheck')
         case "Unit Test":
-            return skip_stage_pragma('unit-test') ||
+                   if (params.CI_UNIT_TEST) {
+                       println("CI_UNIT_TEST found!")
+                   }
+                   if (skip_stage_pragma('unit-test')) {
+                       println("Unit test skip requested")
+                   }
+                   if (skip_stage_pragma('run-test')) {
+                       println("Run test skip requested")
+                   }
```

Comment:
Debug to be removed?

Reply:
Yes.  On my previous test run, this stage was skipped with out anything in the log to indicate why.